src/changelog.ts

```diff
@@ -236,10 +237,10 @@ export default class Changelog {
       const labels = commit.githubIssue.labels.map(label => label.name.toLowerCase());
 
       if (this.config.wildcardLabel) {
-        // check whether the commit has any of the labels from the learna.json config.
+        // check whether the commit has any of the labels from the learnaon config.
         // If not, label this commit with the provided label
 
-        let foundLabel = Object.keys(this.config.labels).some(label => labels.indexOf(label.toLowerCase()) !== -1);
+        const foundLabel = Object.keys(this.config.labels).some(label => labels.indexOf(label.toLowerCase()) !== -1);
```

Comment:
this whole PR would be much easier to review if we didn't have these constant unrelated changes in it 😞